### ISSUE TITLE
hand evaluation metric

### ISSUE THREAD

**Issue by tmddud20** [jianwang-mpi/egowholemocap#6]
Hello,

First of all, I would like to express my respect for your outstanding research.

I have a question regarding the evaluation of the Egowholebody test dataset using the pretrained model you provided. When I calculated the hand MPJPE and PA-MPJPE, the errors were roughly twice as large as the results presented in your paper.

Could you please clarify whether it is correct to use the pretrained model for calculating hand MPJPE and PA-MPJPE? Additionally, I would like to know if you could provide the evaluation metric functions that were used.

Below are the results I obtained during my evaluation:

Left Hand
MPJPE: 76.65
PA-MPJPE: 35.56

Right Hand
MPJPE: 75.64
PA-MPJPE: 35.24

Thank you for your time and assistance.

**Comment by jianwang-mpi** [comment]
Hello,

The released pretrained model is finetuned on the SceneEgo dataset, so it does not work well on the EgoWholeBody test dataset. For the results on the EgoWholeBody dataset, you should finetune the hand4whole model on the EgoWholeBody train dataset.

If I remember correctly, the test evaluation config script I was using is: https://github.com/jianwang-mpi/egowholemocap/blob/main/configs/egofullbody/egofullbody_test_synthetic_fisheye.py

The model file after fine-tuning is: https://nextcloud.mpi-klsb.mpg.de/index.php/s/c3fyD8mm5TfenPa

The log file of the evaluation script is: https://nextcloud.mpi-klsb.mpg.de/index.php/s/kz3BdeRnMCreEdW

I did not have enough time to fully refine the hyper-parameters on the EgoWholeBody train dataset. If you carefully train it yourself, you may have better results than me.

Best,

Jian Wang

**Comment by tmddud20** [comment]
Thank you!
